src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayTariff = 40;
+  const total = days * dayTariff;
+
+  if (days >= 3 && days < 7) {
+    return total - 20;
+  } else if (days >= 7) {
+    return total - 50;
+  }
```

Comment:
1. Read checklist item 3 about `if/else` and `return`
2. Introduce variables for discount values.

Reply:
@VinnieJ-2k20 , Addressed your comments. Please check again.